openmc/mgxs/mgxs.py

```diff
@@ -4615,3 +4628,728 @@ def get_pandas_dataframe(self, groups='all', nuclides='all',
+    tallies : collections.OrderedDict
+        OpenMC tallies needed to compute the multi-group cross section. The keys
+        are strings listed in the :attr:`Velocity.tally_keys` property and
+        values are instances of :class:`openmc.Tally`.
+    rxn_rate_tally : openmc.Tally
+        Derived tally for the reaction rate tally used in the numerator to
+        compute the multi-group cross section. This attribute is None
+        unless the multi-group cross section has been computed.
+    xs_tally : openmc.Tally
+        Derived tally for the multi-group cross section. This attribute
+        is None unless the multi-group cross section has been computed.
+    num_subdomains : int
+        The number of subdomains is unity for 'material', 'cell' and 'universe'
+        domain types. This is equal to the number of cell instances
+        for 'distribcell' domain types (it is equal to unity prior to loading
+        tally data from a statepoint file).
+    num_nuclides : int
+        The number of nuclides for which the multi-group cross section is
+        being tracked. This is unity if the by_nuclide attribute is False.
+    nuclides : Iterable of str or 'sum'
+        The optional user-specified nuclides for which to compute cross
+        sections (e.g., 'U-238', 'O-16'). If by_nuclide is True but nuclides
+        are not specified by the user, all nuclides in the spatial domain
+        are included. This attribute is 'sum' if by_nuclide is false.
+    sparse : bool
+        Whether or not the MGXS' tallies use SciPy's LIL sparse matrix format
+        for compressed data storage
+    loaded_sp : bool
+        Whether or not a statepoint file has been loaded with tally data
+    derived : bool
+        Whether or not the MGXS is merged from one or more other MGXS
+    hdf5_key : str
+        The key used to index multi-group cross sections in an HDF5 data store
+
+    """
+
+    def __init__(self, domain=None, domain_type=None,
+                 groups=None, by_nuclide=False, name=''):
+        super(Velocity, self).__init__(domain, domain_type,
+                                              groups, by_nuclide, name)
+        self._rxn_type = 'velocity'
+
+    @property
+    def scores(self):
+        return ['inverse-velocity', 'flux']
+
+    @property
+    def rxn_rate_tally(self):
+        if self._rxn_rate_tally is None:
+            self._rxn_rate_tally = self.tallies['flux']
+            self._rxn_rate_tally.sparse = self.sparse
+        return self._rxn_rate_tally
+
+    @property
+    def xs_tally(self):
+
+        if self._xs_tally is None:
+            inverse_velocity = self.tallies['inverse-velocity']
+
+            # Compute the velocity
+            self._xs_tally = self.rxn_rate_tally / inverse_velocity
+            super(Velocity, self)._compute_xs()
+
+        return self._xs_tally
+
+    def print_xs(self, subdomains='all', nuclides='all', xs_type='macro'):
```

Comment:
It could, but the units would be listed as cm^-1, which might be confusing. Other than changing line 5214, it is the same as MGXS.print_xs(). One option is to have a `units` attribute for the MGXS class that is used in constructing the header. I could see other MGXS being created down the road that would have other units so this might be useful in more than just the velocity. Any objections to adding a units attribute? If not, I'll go ahead and add it.